server_env_connector_jira/models/jira_backend.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2016-2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import logging
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class JiraBackend(models.Model):
+    _name = "jira.backend"
+    _inherit = ["jira.backend", "server.env.mixin"]
```

Comment:
we should probably use the techname mixin but is not a blocker

Reply:
I already add `server.env.techname.mixin` into inherit file